packages/amplify-frontend-javascript/index.js

```diff
@@ -37,7 +37,8 @@ async function createFrontendConfigsAtPath(context, amplifyResources, amplifyClo
   const cloudOutputsForFrontend = amplifyCloudResources.outputsForFrontend;
 
   const amplifyConfig = await getAWSExports(context, newOutputsForFrontend, cloudOutputsForFrontend);
-  await generateAwsExportsAtPath(context, path.join(exportPath, constants.exportsFilename), amplifyConfig);
+  await generateAwsExportsAtPath(context, path.join(exportPath, constants.exportsJSFilename), amplifyConfig);
+  await generateAwsExportsAtPath(context, path.join(exportPath, constants.exportsJSONFilename), amplifyConfig);
```

Comment:
is this change correct ?
if `generateAwsExportsAtPath` is rendering `aws_exports.js.ejs` template, why would we do that to `json` file ?

Reply:
Updated. VScode decided not to save file.